psi4/driver/p4util/fchk.py

```diff
@@ -0,0 +1,128 @@
+#
+# @BEGIN LICENSE
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2019 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This file is part of Psi4.
+#
+# Psi4 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+"""Module with utility functions for FCHK files."""
+
+import numpy as np
+from psi4.driver.p4util.testing import compare_strings, compare_arrays, compare_values, compare_integers
+from psi4 import core
+from .exceptions import ValidationError
+
+__all__ = ['fchkfile_to_string','compare_fchkfiles']
```

Comment:
Is this necessary? Or is it to stop some import clash elsewhere?

Reply:
Not having it does not break anything but LGTM complained about polluting the namespace. These are the only two functions that should be called directly by users.

Reply:
I see, so it's good practice to declare `__all__` whenever the module is imported via `from module import *`. You learn something every day!